mod_reforged/hooks/entity/tactical/humans/conscript.nut

```diff
@@ -19,12 +19,13 @@
 		this.setAppearance();
 		this.getSprite("socket").setBrush("bust_base_southern");
 		// this.m.Skills.add(this.new("scripts/skills/perks/perk_brawny"));
-		this.m.Skills.add(this.new("scripts/skills/perks/perk_nimble"));
+		// this.m.Skills.add(this.new("scripts/skills/perks/perk_nimble"));
 		// this.m.Skills.add(this.new("scripts/skills/perks/perk_fast_adaption"));
 		// this.m.Skills.add(this.new("scripts/skills/actives/rotation")); // Replaced with perk
 		// this.m.Skills.add(this.new("scripts/skills/actives/recover_skill"));	// Now granted to all humans by default
 
 		// Reforged
+		this.m.Skills.add(this.new("scripts/skills/perks/perk_dodge"));
```

Comment:
I disagree with the addition of Dodge to Conscripts. I'd rather them retain Nimble and lose more defenses than gain Dodge. They are extremely difficult to hit right now. I strongly think that for "regular" enemies it is a more fun design for the player if things are hittable even if it takes more hits to kill compared to missing most of the time.

I'd be ok with them losing Nimble and gaining Poise (I like how the southern faction is "different" from the northern one in their fighting style). The south is still a harsh land with tribal mechanics so I suppose people there have more fighting experience by default - so the conscripts are not necessarily inexperienced imo.

But I am strongly opposed to any defense increases on Conscripts. In a battle the Gunners apply Overwhelm, while Conscripts use Shieldwall with Phalanx - and they are unhittable.

Conscript Polearm also needs to lose Long Reach otherwise the hit-chance Conscripts have even against shielded bros is quite high because they have Backstabber which synergizes very strongly with Long Reach.

Reply:
Better to discuss on voice

Reply:
We decided to:
- Drop Melee Defense and Melee Skill by 10 each compared to vanilla.
- Drop Ranged Defense to 0.
- Remove Nimble, add Poise.
- Not add Dodge.
- Remove Long Reach from conscript polearm.